omod/src/main/webapp/fragments/patientsearch/covid/patientSearchWidget.gsp

```diff
@@ -219,7 +219,7 @@ body {
 <script>
     function searchOnLine(identifier) {
         jq("#loading-model").modal("show");
-        jq.post("https://apitest.cphluganda.org/diagnostic_report",
+        jq.get("https://apitest.cphluganda.org/diagnostic_report",
```

Comment:
make the URL a global property so that it can be changed when the system being searched changes

Reply:
@Mwanje this comment needs your attention